tests/load_interrupt/run.sh

```diff
@@ -60,7 +60,7 @@ function run() {
     check_row_count 2
 
     # only failed at the first two time, will retry later and success
-    export GO_FAILPOINTS='github.com/pingcap/dm/loader/LoadExecCreateTableFailed=return("1152,2")'
+    export GO_FAILPOINTS='github.com/pingcap/dm/loader/LoadExecCreateTableFailed=3*return("1105")'
```

Comment:
why do not inject two variables now?

Reply:
@amyangfei told me about this, I think may be second variable is useless

Reply:
the second is used as failure time, which includes one `create schema` and two `create table` failure

Reply:
so, is it still needed?

Reply:
not needed